### PR TITLE
Niloofar Sadeghi / convert stocks to ts

### PR DESCRIPTION
**Pull request by niloo-fs**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/landing/stocks/_dtrading.tsx [hunk 1]
```diff
@@ -165,14 +196,6 @@ const DTrading = ({ contentMargin, trading, reverse, setWidth, two_title }) => {
...
                                 <StyledHeader type="display-title">{item.title}</StyledHeader>
                                 <Text>{item.subtitle}</Text>
-                                {two_title && (
-                                    <>
-                                        <StyledHeader type="page-title" mt="2.4rem">
-                                            {item.second_title}
-                                        </StyledHeader>
-                                        <Text>{item.second_subtitle}</Text>
-                                    </>
-                                )}
```
Comment (sean-binary) [inline src/pages/landing/stocks/_dtrading.tsx:175]: Not being used?

File: src/pages/landing/stocks/_footerBanner.tsx [hunk 2]
```diff
@@ -128,19 +126,33 @@ const StyledHeaderSmall = styled(Header)`
...
+        display: none;
+    }
+`
 
+type FooterBannerProps = {
+    background_pattern: string
+    is_ppc: boolean
+    title: ReactElement
+    small_title: ReactElement
+    data: { stocks_banner: any }
```
Comment (sean-binary) [inline src/pages/landing/stocks/_footerBanner.tsx:146]: Is stocks_banner an image data? If it is for used for Gatsby Image, you can use the `ImageDataLike` type from `gatsby-plugin-image`.

File: src/pages/landing/stocks/_whyTradeWIthUs.tsx [hunk 3]
```diff
@@ -101,7 +112,6 @@ const WhyTradeWithUs = ({ itemsArr, mainTitle, columnPerRow }) => {
                                 >
                                     {item.title}
                                 </StyledIconTitle>
-                                {item.desc && <Text align="center">{item.desc}</Text>}
```
Comment (sean-binary) [inline src/pages/landing/stocks/_whyTradeWIthUs.tsx:104]: Not in used?

File: src/pages/landing/stocks/_dtrading.tsx (hunk 1 above)
Comment (niloo-fs) [reply src/pages/landing/stocks/_dtrading.tsx:175]: No it was an extra code inside that component.

File: src/pages/landing/stocks/_whyTradeWIthUs.tsx (hunk 3 above)
Comment (niloo-fs) [reply src/pages/landing/stocks/_whyTradeWIthUs.tsx:104]: No, It's not.

**Review by sean-binary — APPROVED** [review]

**Pull request merged**